roles/user/templates/config.j2

```diff
@@ -1 +1,5 @@
+{% if user_sudoers_type == "group" %}
 %{{ item.name }} {{ user_sudoers }}
+{% elif user_sudoers_type == "user" %}
+{{ item.name }} {{ user_sudoers }}
```

Comment:
after reading the `sudoers` man page, I think we only want this variant without any option?

Reply:
That's how it was initially. I added the parameter so that you can use both. I would change the default to user.